.spi.yml

```diff
@@ -3,5 +3,8 @@ metadata:
   authors: Apple Inc.
 builder:
   configs:
-    - documentation_targets: [Testing]
+    - swift_version: 5.10
```

Comment:
We're planning to drop 5.10 support shortly (see #467.) Would we remove these two lines then?

Reply:
Yes, just change it to

```
builder:
  configs:
    - swift_version: 6.0
      documentation_targets: [Testing]
      scheme: Testing
```

then! (And drop the `swift_version: 6.0` line once Swift 6.0 is released.)

Reply:
Filed #471 to track.